lib/lightning_web/live/project_live/settings.html.heex

```diff
@@ -181,7 +181,7 @@
             module={LightningWeb.Components.ProjectDeletionModal}
             id={@project.id}
             project={@project}
-            save_return_to={~p"/"}
```

Comment:
Could the redirect plug be responsible for choosing between Lightning or Thunderbolt project dashboard LV?